src/hs_ontology_api/instance/app.cfg.example

```diff
@@ -9,3 +9,16 @@ ROWLIMIT=1000
 # Maximum size of payload, in bytes
 # 9437184 bytes = 9 MB, less than the 10 MB payload limit for the AWS API gateway.
 PAYLOADLIMIT = 9437184
```

Comment:
In favor of the new S3 handling with using `LARGE_RESPONSE_THRESHOLD`, we should remove the old `PAYLOADLIMIT` from and config.